ui-grid/ui-grid.d.ts

```diff
@@ -2844,7 +2844,7 @@ declare module uiGrid {
              * @param {any} rowEntity gridOptions.data[] array value
              * @param {ng.IAngularEvent} event object if raised from event
              */
-            selectRow(rowEntity: IGridRow, event?: ng.IAngularEvent): void;
+            selectRow(rowEntity: uiGrid.IGridRow, event?: ng.IAngularEvent): void;
```

Comment:
selectRow actually takes an object in the gridOptions.data array as parameter, not uiGrid.IGridRow.